docs/content/5.providers/directus.md

```diff
@@ -0,0 +1,59 @@
+# Directus
+
+Nuxt Image with Directus integration
+
+Integration between [Directus](https://directus.io/) and the image module.
+
+No specific configuration is required. You just need to specify `directus` in your configuration to make it the default provider:
+
+```ts [nuxt.config.ts]
+export default defineNuxtConfig({
+  image: {
+    directus: {}
+  }
+})
+```
+
+You can easily override default options:
+
+```ts [nuxt.config.ts]
+export default defineNuxtConfig({
+  image: {
+    directus: {
+      baseURL: 'http://mydirectus-domain.com/assets/',
+      modifiers: {
+        withoutEnlargement: 'true',
+        transforms: '[["blur", 4], ["negate"]]'
+      }
+    }
+  }
+})
+```
+
+## Modifiers
+All the default modifiers from [Directus documentation](https://docs.directus.io/reference/files.html#requesting-a-thumbnail) are available.
+
+```vue
+<nuxt-img
+  provider="directus"
+  src="ad514db1-eb90-4523-8183-46781437e7ee"
+  height="512"
+  fit="inside"
+  quality="20"
+  :modifiers="{ withoutEnlargement: 'true' }"
+/>
+```
+
+Since Directus exposes [the full sharp API](https://sharp.pixelplumbing.com/api-operation) through the `transforms` parameter, we can use it inside our `modifiers` prop:
+
+```vue
+<nuxt-img
+  provider="directus"
+  src="ad514db1-eb90-4523-8183-46781437e7ee"
+  :modifiers="{ withoutEnlargement: 'true', transforms: '[[&quot;blur&quot;, 4], [&quot;negate&quot;]]' }"
+/>
+```
+
+::alert{type="info"}
+The `transforms` parameter accepts a two-dimensional JSON array. This means that we should either pass an object to our `modifiers` prop or use `&quot;` to wrap around our parameters in the third quotation layer.
+::
```

Comment:
would it make sense to pass an array in here and normalise in `getImage` instead?

Reply:
Probably yes, this was one of those things that I didn't much tuch because of lack of experience in js/ts. Leaving it more barebones for the *experts* to decide.

Reply:
you're doing a great job!

Reply:
later I'll push couple of commits to potentially solve this

Reply:
I'm having an hard time, but I'm starting to think that I forgot how to do the request to Directus correctly, because in a dev test I spun up if I do `http://localhost:8055/assets/ad514db1-eb90-4523-8183-46781437e7ee?transforms=%5B"blur"%2C"4"%5D` I get:

```json
{
    "errors": [
        {
            "message": "\"transforms\" Parameter does not allow \"b\" as a transformation.",
            "extensions": {
                "code": "INVALID_QUERY"
            }
        }
    ]
}
```

and I don't understand why I get `\"b\" instead of blur...

Reply:
Ok I've figured out how to do it, and it has been quite a nice challenge to face. But I'm happy that it fully works.

I'm resolving this with [51edbd7](https://github.com/nuxt/image/pull/787/commits/51edbd77a72e44a0334d7fa1c56b49e4c7cc446d)